package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "dependencies": {
+    "@types/react": "^18.0.25",
+    "i18next": "^22.0.6",
+    "i18next-browser-languagedetector": "^7.0.1",
+    "react-i18next": "^12.0.0"
+  }
+}
```

Comment:
This should be inside the existing `package.json` file inside the 'web` directory.

Just add these to the file:
```
    "i18next": "^22.0.6",
    "i18next-browser-languagedetector": "^7.0.1",
    "react-i18next": "^12.0.0"
```
and run `npm i`

Reply:
This file along with the lock file should then be deleted.